Common/Securities/Option/OptionFilterUniverse.cs

```diff
@@ -224,6 +225,483 @@ public OptionFilterUniverse PutsOnly()
         {
             return Contracts(contracts => contracts.Where(x => x.ID.OptionRight == OptionRight.Put));
         }
+
+        /// <summary>
+        /// Sets universe of a single call contract with the closest match to criteria given
+        /// </summary>
+        /// <param name="daysTillExpiry">The desire days till expiry from the current time</param>
+        /// <param name="strikeFromAtm">The desire strike price distance from the current underlying price</param>
+        /// <remarks>Applicable to Naked Call, Covered Call, and Protective Call Option Strategy</remarks>
+        /// <returns>Universe with filter applied</returns>
+        public OptionFilterUniverse SingleCall(int daysTillExpiry = 30, decimal strikeFromAtm = 0)
+        {
+            return SingleContract(OptionRight.Call, daysTillExpiry, strikeFromAtm);
+        }
+
+        /// <summary>
+        /// Sets universe of a single put contract with the closest match to criteria given
+        /// </summary>
+        /// <param name="daysTillExpiry">The desire days till expiry from the current time</param>
+        /// <param name="strikeFromAtm">The desire strike price distance from the current underlying price</param>
+        /// <remarks>Applicable to Naked Put, Covered Put, and Protective Put Option Strategy</remarks>
+        /// <returns>Universe with filter applied</returns>
+        public OptionFilterUniverse SinglePut(int daysTillExpiry = 30, decimal strikeFromAtm = 0)
```

Comment:
All of these strategy helpers are `Single`, should we just rename this to `NakedPut` & `NakedCall` helper methods?